components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -7192,6 +7192,22 @@ public static JSONObject getTenantRESTAPIScopeRoleMappingsConfig(String tenantDo
         return restAPIConfigJSON;
     }
 
+    public static String getGAConfig(String tenantDomain) throws APIManagementException {
```

Comment:
shall we rename this method to a more meaningfull one?

Reply:
Changed